src/test/resources/com/arpnetworking/logback/StenoEncoderTest.testEncodeArrayWithException.json

```diff
@@ -1 +1 @@
-{"time":"1970-01-01T00:00:00.000Z","name":"logEvent","level":"info","data":{},"exception":{"type":"java.lang.NullPointerException","message":"npe!","backtrace":[],"data":{}},"context":{"host":"<HOST>","processId":"<PROCESS_ID>","threadId":"<THREAD_ID>"},"id":"<ID>","version":"0"}
+{"time":"1970-01-01T00:00:00.000Z","name":"logEvent","level":"info","data":{},"exception":{"type":"java.lang.NullPointerException","message":"npe!","backtrace":[],"data":{"_id":"<ID>","_class":"java.lang.NullPointerException"}},"context":{"host":"<HOST>","processId":"<PROCESS_ID>","threadId":"<THREAD_ID>"},"id":"<ID>","version":"0"}
```

Comment:
I was conflicted on this. However, I went with this route because the exception data block serialization is consistent with bean (data) serialization. 

That is, a non-@Loggable, non-@LogValue-providing exception instance will serialize to an id and class; in such a case the type field is part of the Steno format duplicates the class name. However,  if the exception were @Loggable the class field would not be logged. 

So we either get type-class duplication in some cases or we get inconsistency between how bean's are serialized and how exception's are serialized.

Thoughts?